src/cli/create.ts

```diff
@@ -86,59 +96,55 @@ export async function create (destDir: string | undefined, opts: { template?: st
             // Continue
         }
 
-        template = opts.template || await prompt({
+        const template = opts.template || await prompt({
             type: "select",
             name: "template",
             message: "Choose a project template below",
             choices: [
-                { title: "Daily Level Game", value: "daily-level" },
+                { title: "Daily Level Game", value: "github:playpassgames/playpass-game-template" },
                 { title: "React Daily Level Game", value: "react-daily-level" },
-                { title: "Daily Word Game", value: "word-daily-level" },
+                { title: "Daily Word Game", value: "github:playpassgames/daily-word-game-template" },
+                { title: "Daily Phrase Game", value: "github:playpassgames/daily-phrase-game-template" },
             ],
         });
-    }
-
-    console.log();
-
-    let game: Game;
-    try {
-        game = await playpassClient.create(subdomain);
-    } catch (e) {
-        throw new Error(`Failed to create game ${subdomain}, please try again`);
-    }
 
-    if (useTemplate) {
-        const transform = (src: string, dest: string) => {
-            const ext = path.extname(src);
-            if (ext != ".js" && ext != ".jsx" && ext != ".ts" && ext != ".tsx") {
-                return null;
-            }
-            return replace("YOUR_GAME_ID", game.id);
-        };
-
-        const templatesDir = `${__dirname}/../../../templates`;
-        await copy(`${templatesDir}/${template}`, destDir, { dot: true, junk: true, transform });
-        await copy(`${templatesDir}/common`, destDir, { dot: true, junk: true });
+        // values with slashes in them are preceived as git urls
```

Comment:
would `template.indexOf("github:")` be more appropriate here?

Reply:
I believe there's product intent to allow any git url so that we may support third parties creating templates for our platform in the future.  That's why I didn't restrict it to just `github:playpassgames/`